File: Lab-1/1_3/src/Die.java
//Java Program by Alex Bledsoe, 04/07/2017

import java.util.Random;

class Die {
    private Random rand = new Random();
    private int currentNumber;

    Die() {
        roll();
    }

    private void roll() {
        currentNumber = rand.nextInt(6) + 1;
    }

    int getCurrentNumber() {
        return currentNumber;
    }
}

File: KotlinConversion/src/Lab_1/Num1_3/Die.kt
package Lab_1.Num1_3

import java.util.*

/**
 * Created by Alex Bledsoe, 6/2/2017.
 */
class Die {
    private val rand = Random()
    private var currNum: Int = 0

    init {
        roll()
    }

    fun roll() {
        currNum = rand.nextInt(6) + 1
    }

    fun getCurrNum(): Int {
        return currNum
    }
}